docs-new/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why is there a LICENSE file? Wouldn't this be covered by the LICENSE in the root of the repository?

Reply:
Yup I'll remove